src/gennylib/include/gennylib/context.hpp

```diff
@@ -544,6 +544,16 @@ class PhaseContext final : public v1::HasNode {
             internal);
     }
 
+    auto namedOperation(const std::string& metricsName, ActorId id, bool internal = false) const {
```

Comment:
Let's mark this as experimental in a comment or at least write a good doc about why this was necessary; I think this API is covering up a shortcoming in the original context APIs.

Reply:
[TIG-4328](https://jira.mongodb.org/browse/TIG-4328)